text/0000-payment-proofs.md

```diff
@@ -0,0 +1,101 @@
+- Title: payment-proofs
+- Authors: [David Burkett](mailto:davidburkett38@gmail.com)
+
+---
```

Comment:
Could this be replaced with the header of the RFC template?

https://github.com/mimblewimble/grin-rfcs/blob/master/0000-template.md